Gemfile

```diff
@@ -30,7 +30,7 @@ group(:development, :test) do
   # gem 'chefstyle', github: 'chef/chefstyle'
   gem "chefstyle", git: "https://github.com/chef/chefstyle.git", branch: "master"
 
-  gem "ruby-shadow", platforms: :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
+  gem "ruby-shadow", platforms: :ruby unless RUBY_PLATFORM.downcase =~ /(aix|cygwin)/
```

Comment:
Personally, I think =~ is a bit more intention-revealing, but it's a bit of a wash.